source/_integrations/ssdp.markdown

```diff
@@ -25,6 +25,7 @@ ssdp:
 The following integrations are automatically discovered by the SSDP integration:
 
  - [deCONZ](../deconz/)
+ - [DirecTV](/integrations/directv/)
```

Comment:
```suggestion
 - [DirecTV](../directv/)
```

Reply:
No, we should full links from the root of the project. `/integrations/directv` is correct.